apps/docs/src/components/Dashboard/Usage.tsx

```diff
@@ -0,0 +1,106 @@
+import { useEffect, useState } from 'react'
+import { Card, CardContent } from '../ui/card'
+import { LoaderIcon } from 'lucide-react'
+import LineChart from './Chart'
+
+const usageUrl = `${process.env.NEXT_PUBLIC_BILLING_API_URL}/teams/usage`
+
+export const UsageContent = ({ currentApiKey }: { currentApiKey: string | null}) => {
+  const [vcpuData, setVcpuData] = useState<any>(null)
+  const [ramData, setRamData] = useState<any>(null)
+
+  useEffect(() => {
+    const getUsage = async () => {
+      setVcpuData(null)
+      setRamData(null)
+
+      const response = await fetch(usageUrl, {
+        headers: {
+          'X-Team-API-Key': currentApiKey! 
+        }
+      })
+      if (!response.ok) {
+        // TODO: Add sentry event here
+        console.log(response)
+        return
+      }
+
+      const data = await response.json()
+
+      const { vcpuSeries, ramSeries } = transformData(data.usages)
+      
+      setVcpuData(vcpuSeries)
+      setRamData(ramSeries)
+    }
+    if (currentApiKey) {
+      getUsage()
+    }
+  }, [currentApiKey])
+
+  return (
+    <div className="flex flex-col w-full h-full pb-10">
+      <div className='pb-10'>
+        <h2 className='font-bold pb-4 text-xl'>Usage history</h2>
+        <p>
+          The graphs show the total monthly vCPU-hours and RAM-hours used by the team. <br/>
+        </p>
+      </div>
+
+      {vcpuData && ramData ? (
```

Comment:
If there's no usage, it's showing `NaN` and there's no `0`. 
<img width="1762" alt="image" src="https://github.com/e2b-dev/E2B/assets/50249709/f82e933f-6f37-4938-88e6-a92e9b3628ef">

Reply:
hmm, could you add the console messages.
Basically either there's an error in getting your data --> no data
Or something goes wrong in changing the shape of the data --> formatting error on client side

Reply:
```json
{
    "credits": 30,
    "usages": [
        {
            "month": 3,
            "template_usage": [],
            "unpaid_cost": 0,
            "year": 2024
        },
        {
            "month": 4,
            "template_usage": [],
            "unpaid_cost": 0,
            "year": 2024
        },
        {
            "month": 5,
            "template_usage": [],
            "unpaid_cost": 0,
            "year": 2024
        },
        {
            "month": 6,
            "template_usage": [],
            "unpaid_cost": 0,
            "year": 2024
        }
    ]
}
```
<img width="1236" alt="image" src="https://github.com/e2b-dev/E2B/assets/50249709/c149222d-7883-494c-b3f4-dd7ace8da6d4">

Reply:
but so the response is that you have no usage..? 🤔

Reply:
Now I think I understand you, this was what you were after? 😄

https://github.com/ErikKaum/E2B/blob/df3eb30bb2404d531f8635398803cbbc80027c75/apps/docs/src/components/Dashboard/Usage.tsx#L75